Assets/MixedRealityToolkit.Examples/Demos/Input/Scenes/TurnPointerOnOffExample/TunrPointersOnOff Configuration Profile.asset

```diff
@@ -0,0 +1,55 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
```

Comment:
I don't think we usually have spaces in our assets (i.e. there's spaces between Configuration/Profile/TunrPointersOnOff.

Also I think there's a misspelling in TunrPointersOnOff -> TurnPointersOnOff

Reply:
Same general comment on spaces for all of the newly added assets.